core/src/rpc.rs

```diff
@@ -270,6 +270,53 @@ impl JsonRpcRequestProcessor {
         Ok(new_response(&bank, response))
     }
 
+    pub fn get_multiple_accounts(
+        &self,
+        pubkeys: Vec<Pubkey>,
+        config: Option<RpcAccountInfoConfig>,
+    ) -> Result<RpcResponse<Vec<Option<UiAccount>>>> {
+        let mut accounts: Vec<Option<UiAccount>> = vec![];
+
+        let config = config.unwrap_or_default();
+        let bank = self.bank(config.commitment);
+        let encoding = config.encoding.unwrap_or(UiAccountEncoding::Binary);
```

Comment:
For new methods wdyt about moving to Base64 as the default?  It's inconsistent but also the direction that we want to go in I think (that is, any direction away from base58)

Reply:
I had the same thought as I was typing that Binary. Yeah, let's do it here.